index.html

```diff
@@ -1,54 +1,273 @@
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button multi">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+      </section>
+
+      <section class="aboutmyself">
+        <h2>About Myself</h2>
+
+        <article class="offer">
+          <p>
+            Hello I’m a software developer! I can
+            help you build a product , feature or
+            website. Look through some of my
+            work and experience! If you like what
+            you see and have a project you need
+            coded, don’t hesitate to contact me.
+          </p>
+        </article>
+        <section class="contact">
+          <p>LET’S CONNECT</p>
+        </section>
+
+        <section class="socialmedia-botton">
+          <ul>
+            <li><img src="./images/git.png" alt=""></li>
+            <li><img class="in" src="./images/Linkedin icon.svg" alt=""> </li>  
+            <li><img class="hand" src="./images/hand.png" alt=""></li>
+            <li class="twitter"><img src="./images/twitter.png" alt=""></li>
+            <li><img class="m" src="./images/m.png" alt=""></li>
+          </ul>
+        </section>
+
+        <div class="get">
+          <a href="#"><button>Get my resume</button></a>
+        </div>
+
+
+        <section class="language">
+
+          <div class="legheader">
+            <ul>
+              <li>
+                Languages
+              </li>
+            </ul>
+            <img class="arrowdown" src="./images/arrow-down.png" alt="">
+          </div>
+
+          <div class="logo">
+            <div class="leng">
+              <div class="image"><img src="./images/Ellipse 1.png" alt=""></div>
+              <p>JavaScript</p>
+            </div>
+              
+            <div class="leng">
+              <div class="image"><img src="./images/Ellipse 1-1.png" alt=""></div>
+              <p>HTML</p>
+            </div>
+
+            <div class="leng">
+              <div class="image"><img src="./images/Ellipse 1-2.png" alt=""></div>
+              <p>CSS</p>
+            </div>
+          </div>
+
+          <div class="frame">
+
+            <div class="work">
+              <ul>
+                <li>
+                  Frameworks
+                </li>
+              </ul>
+              <div class="image"><img src="./images/arrow-right.png" alt=""></div>
+             
+            </div>
+              <hr>
+            <div class="skills">
+              <p>Skills</p>
+              <div class="image"><img src="./images/arrow-right.png" alt=""></div>
+            </div>
+            <hr>
+          </div>
+        </section>
+      </section>
```

Comment:
- [ ] Please consider to use `ul` to wrap the list of  The technology in each category ("Languages", "Frameworks" and "Skills") and make sure each of them is `li`. This will make your code way more cleaner 🧼 and improve the user experience 👍

Mentioned by the previous reviwer:
> * [ ]  Also, please wrap all of the technology categories ("Languages", "Frameworks" and "Skills")  in a `ul` tag, and each of them in a `li` tag. Similarly, please make sure that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`, as it is very necessary for accessibility and semantic purposes. 👍